pre_processing/read_himawari.F90

```diff
@@ -223,7 +270,7 @@ subroutine read_himawari_bin(infile, imager_geolocation, imager_measurements, &
    real(kind=sreal)   :: rye, rhr, rminu
```

Comment:
Just caught an unused variable warning in this routine. What should `tlat, tlon` be doing? They aren't allocated but are flagged for multithreading.

Reply:
I didn't touch these variables in this PR, but good catch. Maybe a question for @simonrp84? Can we delete these?

Reply:
Those variables are needed for speeding up processing with the PGI compilers, so are necessary.